### PR TITLE
fix(ReactTextView): Add support for 'padding' property

### PR DESCRIPTION
**Pull request by ebragge**

### CODE REVIEW

File: ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs [hunk 1]
```diff
@@ -10,7 +10,10 @@ namespace ReactNative.UIManager
     /// </summary>
     public class LayoutShadowNode : ReactShadowNode
     {
-        private const float Undefined = CSSConstants.Undefined;
+        /// <summary>
+        /// Undefined float value.
+        /// </summary>
+        protected internal const float Undefined = CSSConstants.Undefined;
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs]: Any reason we can't delete this constant all together and just inline the reference to CSSConstants.Undefined?

File: ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs [hunk 2]
```diff
@@ -165,7 +168,7 @@ public void SetMargins(int index, float margin)
             ViewProps.PaddingTop,
             ViewProps.PaddingBottom,
             DefaultSingle = Undefined)]
-        public void SetPaddings(int index, float padding)
+        public virtual void SetPaddings(int index, float padding)
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs]: Didn't we avoid making this virtual for TextInput?  I believe we somehow just piggy-backed off the underlying Padding value that was set on CSSNode.

Comment (ebragge) [inline ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs]: If padding is set both for TextBlock  and CSSNode the padding on bottom is set twice.

File: ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs (hunk 1 above)
Comment (ebragge) [inline ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs]: No

File: ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs [hunk 3]
```diff
@@ -121,12 +121,29 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
 
             if (_inline != null)
             {
+                var _paddings = new Thickness()
+                {
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs]: Why are we only applying the left and top paddings?

**Comment by rozele** [comment]
:clock1:

File: ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs (hunk 3 above)
Comment (ebragge) [inline ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs]: The thickness is only used to position the text correctly. The layout takes care of the required dimensions.

**Comment by rozele** [comment]
Please cherry pick your commits and reapply to the rebased ReactWindows branch.

**Pull request merged**